src/pages/priceEntry.tsx

```diff
@@ -94,48 +91,60 @@ const PriceEntry = ({
+                    <span className="govuk-hint" id="price-entry-hint">
+                        Example: £2.40 would be 240
+                    </span>
+                    {errors.length > 0 ? createErrorSpans(errors) : null}
+
+                    <div className="fare-triangle-container" role="table">
+                        <div className="fare-triangle" role="rowgroup">
                             {stageNamesArray.map((rowStage, rowIndex) => (
-                                <div id={`row-${rowIndex}`} className="fare-triangle-row" key={rowStage}>
+                                <div id={`row-${rowIndex}`} className="fare-triangle-row" role="row" key={rowStage}>
+                                    <span
+                                        className="govuk-heading-s fare-triangle-label-left"
+                                        role="rowheader"
+                                        key={stageNamesArray[rowIndex]}
+                                    >
+                                        <span>{rowIndex > 0 ? rowStage : null}</span>
+                                    </span>
                                     {stageNamesArray.slice(0, rowIndex).map((columnStage, columnIndex) => (
                                         <React.Fragment key={columnStage}>
-                                            <input
-                                                className={createClassName(
-                                                    faresInformation,
-                                                    rowIndex,
-                                                    rowStage,
-                                                    columnStage,
-                                                )}
-                                                id={`cell-${rowIndex}-${columnIndex}`}
-                                                name={`${rowStage}-${columnStage}`}
-                                                type="text"
-                                                defaultValue={getDefaultValue(faresInformation, rowStage, columnStage)}
-                                            />
+                                            <span
+                                                role="cell"
+                                                className={`fare-triangle-input ${
+                                                    rowIndex % 2 === 0
+                                                        ? 'fare-triangle-input-white'
+                                                        : 'fare-triangle-input-light-grey'
+                                                }`}
+                                            >
+                                                <input
+                                                    id={`cell-${rowIndex}-${columnIndex}`}
+                                                    className={createClassName(faresInformation, rowStage, columnStage)}
```

Comment:
On error, the `govuk-input--error` class is not being applied to some inputs in the right way.